### PR TITLE
Iter2 todos

### PR DESCRIPTION
**Pull request by Poomon001**
- Clean up the engine by reducing complex parameters
- Update stock_transaction to match wallet_transaction

### CODE REVIEW

**Comment by alexwholland** [comment]
Can you make the pr for the `dev` branch

File: tests/TestRun2/InitialSetup.jmx [hunk 1]
```diff
@@ -71,7 +71,7 @@
           </elementProp>
           <elementProp name="configPath" elementType="Argument">
             <stringProp name="Argument.name">configPath</stringProp>
-            <stringProp name="Argument.value">tests/TestRun2/Config</stringProp>
+            <stringProp name="Argument.value">\tests\TestRun2\Config</stringProp>
```
Comment (alexwholland) [inline tests/TestRun2/InitialSetup.jmx:74]: No need to commit this file

File: tests/TestRun2/UserFiles/UserCreds_10.csv [hunk 2]
```diff
@@ -1,11 +1,101 @@
...
-s2v8g00zbmkqbu,EpUe8@3PyG9U_y0fuLEk,Amber Sullivan
-fejtk07w2f8r5k9y2b4l,iG2OfqTIy9lZA_pGFHdU,Reginald Richards
-zv6d53sdzqzz8pq9pzui,lN9k_4Wooe1YczdHV0ldpD78,Benjamin Perez
-0jweqdqs7t87,u_C9YvXnT52,Amanda Whitehead
-3yqiaj0fd6x87ck3p72,9jiZho31t_8EWaA9Uj964D4l6,Janice Marshall
-nub390xf62m85,ErS4mbvxPg3FS9Cnd_ZeJadqIB8l,Jason Foster
-wkr2f3cok94lkvl,rJR56y@nI7L3ytP,Adam Michael
-8rr5ojjyrr527zz9b9,MquD1CF3@Z3ymD_XWM7sNsuldI,Tina Riggs
-ewurd9bqn1m,wBW26K1IS6@HtCYL5sM3UuLTK,Tyler Miller
+Username,Password,Name
```
Comment (alexwholland) [inline tests/TestRun2/UserFiles/UserCreds_10.csv:1]: We shouldn't change this file, we should make new test files for 100, 1000, 10000

File: tests/TestRun2/UserThreadTest.jmx [hunk 3]
```diff
@@ -71,17 +71,17 @@
           </elementProp>
           <elementProp name="stockFile" elementType="Argument">
             <stringProp name="Argument.name">stockFile</stringProp>
-            <stringProp name="Argument.value">tests/TestRun2/ConfigstockIds.csv</stringProp>
+            <stringProp name="Argument.value">\TestRun2\Config\stockIds.csv</stringProp>
```
Comment (alexwholland) [inline tests/TestRun2/UserThreadTest.jmx:74]: No need to commit this file

**Review by alexwholland — CHANGES_REQUESTED** [review]

File: tests/TestRun2/UserThreadTest.jmx (hunk 3 above)
Comment (Poomon001) [reply tests/TestRun2/UserThreadTest.jmx]: I convert it back. And just ignore file for everyone next push. This can be merged fine

**Review by alexwholland — APPROVED** [review]

**Pull request merged**